README.md

```diff
@@ -58,7 +58,7 @@ To try out the ForgeRock JavaScript SDK please look at one of our samples:
 
 - [**React Todo - `/samples/reactjs-todo`**](./samples/reactjs-todo/README.md)
 
-Since this sample app uses npm's workspaces feature, npm 7 or higher is required. When on npm 7+, dependencies are already installed when running `npm install`. You can run this sample app with: `npm run start:reactjs-todo`.
+- [**Angular Todo - `/samples/angular-todo`**](./samples/angular-todo/README.md)
```

Comment:
I think you accidentally replaced a paragraph with your addition. Can you add the note about needing npm 7 back?

Reply:
Ryan asked me to remove this paragraph

Reply:
Thinking about it, we don't specify in the main README the need to use npm 7. Perhaps we should have it in the **Requirements** section along with the Node version?

Reply:
After discussing with @cerebrl I've now changed the main README to include npm 7 as a requirement